geoips/schema/gridlines.yaml

```diff
@@ -77,5 +76,4 @@ properties:
             default: 1
           spacing:
             type: [number, string]
-            enum: [auto]

```

Comment:
@jsolbrig Note that the "grid_lat_xoffset", etc fields would be used to specify the offset for the lat/lon labels - it is not implemented right now, but I think it is probably something that we will want to support.

Reply:
@mindyls Should we just add that in when we add support for it?

Reply:
See #147